transforms/button-destructive/README.md

```diff
@@ -1,40 +1,40 @@
 # button-destructive
 
-In order to add secondary and tertiary buttons to carbon-react, the `destructive` type has been replaced with a dedicated prop.
+In order to add secondary and tertiary buttons to carbon-react, the `destructive` buttonType has been replaced with a dedicated prop.
```

Comment:
It looks like we replaced type property with buttonType. Therefore following would describe it clearly:
```suggestion
In order to add `secondary` and `tertiary` buttons to carbon-react, the `type` property has been replaced with a dedicated property called `buttonType`.
```

Reply:
Yes, I'll update this in a commit.

Reply:
This PR might look a little confusing because `master` calls it `type`, this was a mistake hence the `fix` commit changing it to `buttonType.